README.md

```diff
@@ -2321,6 +2316,13 @@ ctx.session.maxSchemaAgreementWaitSeconds=1
 ctx.session.addressTranslator=com.datastax.driver.core.policies.IdentityTranslator
 ```
 
+### quill-cassandra-streaming-monix
+
```

Comment:
could you add the sbt dependency example (see https://github.com/getquill/quill/blob/dc18f615d7899519a4a3b1431ba93de244ded38d/README.md#sbt-dependencies-9)

Reply:
yeah, I've missed that, moment